cpp/src/interop/from_arrow.cu

```diff
@@ -238,7 +238,9 @@ std::unique_ptr<column> dispatch_to_cudf_column::operator()<bool>(
   rmm::mr::device_memory_resource* mr)
 {
   auto data_buffer = array.data()->buffers[1];
-  auto data        = rmm::device_buffer(data_buffer->size(), stream, mr);
+  // mask-to-bools expects the mask to be bitmask_type aligned/padded
+  auto data = rmm::device_buffer(
+    cudf::bitmask_allocation_size_bytes(data_buffer->size() * CHAR_BIT), stream, mr);
```

Comment:
Multiplying by `CHAR_BIT` seems to be the purpose of this function: https://github.com/rapidsai/cudf/blob/ad0254566ed9f1cb33338feb156c5751ec3747e4/cpp/include/cudf/utilities/bit.hpp#L45

Reply:
I would agree with you Bradley if `size_in_bits` took a size in bytes argument. But as it is, you have to write `data_buffer->size() * size_in_bits<char>()`, which is kinda dumb.

Reply:
https://github.com/rapidsai/cudf/issues/13587